README.md

```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to Awesome Game of Tetris
```

Comment:
new title

Reply:
Note: The problem here is that too many Tetris pieces are the same color. In your review, you might want to suggest that I choose colors which would be easier to see when playing the game.